### PR TITLE
Managing messages is allowed for chat moderators but moderating users is for moderators

### PR DESCRIPTION
**Pull request by gorciu-official**
I think this will be better

### CODE REVIEW

File: src/plugins/moreUserTags/index.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
  * Vencord, a modification for Discord's desktop app
- * Copyright (c) 2022 Vendicated and contributors
+ * Copyright (c) 2022-2024 Vendicated and contributors
```
Comment (Sqaaakoi) [inline src/plugins/moreUserTags/index.tsx:3]: almost nobody changes this, though it isn't a bad thing that this has ben changed

**Review by Sqaaakoi — COMMENTED** [review]
your changes seem completely uncontroversial to me

File: src/plugins/moreUserTags/index.tsx [hunk 2]
```diff
@@ -100,7 +100,7 @@ const tags: Tag[] = [
         name: "CHAT_MODERATOR",
         displayName: "Chat Mod",
         description: "Can timeout people",
```
Comment (Vendicated) [inline src/plugins/moreUserTags/index.tsx:102]:
```suggestion
        description: "Can manage messages",
```

File: src/plugins/moreUserTags/index.tsx [hunk 3]
```diff
@@ -90,7 +90,7 @@ const tags: Tag[] = [
         name: "MODERATOR",
         displayName: "Mod",
         description: "Can manage messages or kick/ban people",
```
Comment (Vendicated) [inline src/plugins/moreUserTags/index.tsx:92]:
```suggestion
        description: "Can kick, ban, or timeout people",
```

**Review by Vendicated — COMMENTED** [review]
you forgot to update the descriptions

File: src/plugins/moreUserTags/index.tsx (hunk 1 above)
Comment (Vendicated) [reply src/plugins/moreUserTags/index.tsx:3]: do not

**Pull request closed without merging**